app/controllers/merchant/dashboard_controller.rb

```diff
@@ -1,4 +1,7 @@
 class Merchant::DashboardController < Merchant::BaseController
   def index
+      current_user
+      @merchant = Merchant.find(current_user.merchant_id)
+      @orders = @merchant.orders.joins(:item_orders)
```

Comment:
What does the `.joins(:item_orders)` do here? I just tried it pry and I get the same value with just `@merchant.orders`

Also it looks like we are not using `@orders` in the dashboard view. Do we need it?

Reply:
You're right I didn't catch that. I spent all that time figuring that out when I didn't even use it  🤦